### ISSUE TITLE
adding acpid support?

### ISSUE THREAD

**Issue by charliedrage** [rancherio/os#103]
I know this is suppose to be minimalistic, but in order to respond to virtualbox / kvm / etc commands. Would you perhaps consider implementing apcid support?

**Comment by wlan0** [comment]
@charliedrage We would be interested in adding acpid support. At this point, I'm not sure if we would package it along with the OS or allow users to add it as a system container. For option 2, We are working on a framework to allow users to add system containers.

**Comment by charliedrage** [comment]
Aha. That'd do it. I'm mainly interested in the bare-metal version of Rancher (installing it on a cloud or your own node). Care to point me in the right direction on how RancherOS is written to bare-metal? I'm also trying to change the Grub boot time (it's current set to 3 seconds) to 1 second for faster boot speed.

**Comment by ibuildthecloud** [comment]
@charliedrage I'll be honest, the bare metal support is very poor.  This should be addressed this week.  There are two main issues

1) We only compiled in drivers for virtual hardware.  Even something as basic as USB is not even enabled.
2) We don't have udev hotplugging so firmware loading fails.

We should have both of these issue addressed shortly.  The installation to disk is documented here https://github.com/rancherio/os#installing-to-disk which internally uses https://github.com/rancherio/os-installer.

For bare metal you don't have to install to disk if you can PXE boot.  In the PXE boot case you just need a partition with LABEL=RANCHER_STATE formatted for persistence.

**Comment by charliedrage** [comment]
@ibuildthecloud 

Found the os-installer! Already made a pull request earlier for changing the grub-loading time. In all, excellent work, I found it so useful. To note, I work for a Cloud company and I've had three users so far in the time I've worked there asking for rancher to be installed, I'm looking forward in the future to the 1.0 launch.

I got rancher installed correctly on a bare-metal node for testing purposes, as well as KVM.

To note, I only mentioned bare-metal since I want to try and get it as minimalist as possible without the overhead of KVM virtualization. I understand that rancher is mainly a VM/Cloud OS not a bare-metal one.
